tests/test_eval.py

```diff
@@ -132,3 +132,33 @@ def test_multi_protein_single_ligand_scoring(system_1ai5, predicted_pose_1ai5):
             assert np.isclose(scores[k], true_scores[k])
         else:
             assert scores[k] == true_scores[k]
+
+
+def test_write_docking_eval(plinder_src, prediction_csv):
+    import subprocess
+    import pandas as pd
+    cmd1 = "plinder_eval" + \
+            f" --prediction_file {prediction_csv} --data_dir "+\
+            f"{plinder_src}/tests/test_data/eval --output_dir "+\
+            f"{prediction_csv.parent}  --num_processes 65"
+    p1 = subprocess.Popen(
+            cmd1, shell=True,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE)
+    p1.communicate()
+
+    score_df = pd.read_parquet(f"{prediction_csv.parent}/scores.parquet")
+    assert np.allclose(score_df.scrmsd_wave.to_list(), [1.617184, 3.665143])
```

Comment:
Seems from failing test that you may need to sort by reference_system_id here first

Reply:
At least 7 tests are still failing on my end.

Reply:
? There's only one test failing in CI, but making updates for the conversion to `plinder_eval` and `plinder_stratify`